ansys/mapdl/core/launcher.py

```diff
@@ -611,7 +611,7 @@ def launch_mapdl(exec_file=None, run_location=None, jobname='file',
                  nproc=2, ram=None, mode=None, override=False,
                  loglevel='ERROR', additional_switches='',
                  start_timeout=120, port=None,
-                 custom_bin=None, cleanup_on_exit=True,
```

Comment:
This is causing https://github.com/pyansys/pymapdl/issues/493

Reply:
Resolved in latest release.